### DISCUSSION TITLE
Mazhar Bhai Ap Ny Bary Mazy Ki App Banai Hai

### DISCUSSION THREAD

**Discussion by 0sama-Mirza** [realmazharhussain/gdm-settings#161] [General]
Aj donation ka message aaya tha. Pr mery pas pasy nahi hn. Mujy pata nahi tha k Pakistani ny ye application banai hai. Kisi din mein bhi kuch na kuch contribute karn ga.

Shukria

**Comment by realmazharhussain** [comment]
Thank you very much for your kind words.

It is good to know that someone from my country is also using my app. And it is a pleasure to read Urdu. But I think it may feel non-inclusive to other users of the app who engage in GitHub discussions. So, I would prefer if you used English in the official channels. However, I would love some Urdu in personal chats if you are inclined. My social links are available on [my GitHub profile](https://github.com/realmazharhusssain). 

As for the donation situatuin, monetary donations are only for people who have enough money that donating does not feel a significant charge on their bank account.

There are other ways to contribute if you really want to. For example translating, doing artwork for the project, commenting something useful in the issue tracker, helping other users, etc.
